test/benchmarks/benchmark_chunkmemset.cc

```
@@ -0,0 +1,127 @@
+        }
+        for (size_t i = 0; i < BUFSIZE; i++) {
+            buf[i] = (uint8_t)(i & 0xFF);
+        }
+    }
+
+    void TearDown(const ::benchmark::State&) {
+        zng_free_aligned(buf);
+    }
+
+    // Bench: state.range(0) = dist, state.range(1) = len
+    void Bench(benchmark::State& state, chunkmemset_safe_func fn) {
```

Cppcheck (2.20.0)

[style] The function 'Bench' is never used.

(unusedFunction)